tests/test-switchroot.sh

```diff
@@ -56,7 +56,7 @@ find_in_env() {
 		"$1" "$tmpdir"
 		enter_fs "$tmpdir"
 		ostree-prepare-root /sysroot
-		find /
+		find / \( -path /proc -o -path /sysroot/proc \) -prune -o -print
```

Comment:
OK with this, but is it really just `/proc`?  Wouldn't e.g. `/sys` and other filesystems be there too?  Oh, I see the script just mounts that.

Reply:
I should've added a comment in there - because processes from outside are visible, recursing down into those proc directory fails when process die.